lib/hz2600/Kazoo/Api/Entity/Device.php

```diff
@@ -4,9 +4,10 @@
 
 class Device extends AbstractEntity
 {
-    public function quickcall($number) {
+    public function quickcall($number, $auto_answer = true) {
```

Comment:
Thanks!  This is a nice addition however I propose a modification let me know what you think!

There are a few properties that are available in addition to auto_answer (such as cid-name/number, timeout, ect).  If the second argument was an array of parameters then that would allow control and support future enhancements without growing the function arg count. 

For example:
public function quickcall($number, $parameters) {
    $query_string = "";
    if (!empty($parameters)) {
       $query_string = "?" . http_build_query($parameters);
    }
    ...
    $this->get(array(), '/quickcall/{quickcall_number}' . $query_string);
}


That would make the usage:

$parameters = array(
       'auto_answer' => true,
       'timeout' => 30,
       'cid-number' => '+14158867900'
);
$sdk->Device->quickcall('1002', $parameters);